composer.json

```diff
@@ -34,7 +35,7 @@
             "Contao\\ManagerPlugin\\Composer\\AppAutoloadPlugin"
         ],
         "symfony": {
-            "require": "^3.3 || ^4.0 || ^5.0"
+            "require": "3.4.* || 4.4.* || ^5.0"
```

Comment:
In case you wonder, this is just an optimization for symfony/flex. Allowing `^3.3 || ^4.0 || ^5.0` increases the resolving time noticeably.